src/main/twirl/uk/gov/hmrc/sca/views/NewScaLayout.scala.html

```diff
@@ -0,0 +1,187 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.auth.core.retrieve.v2.TrustedHelper
+@import uk.gov.hmrc.hmrcfrontend.views.config.StandardAlphaBanner
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.hmrcstandardpage._
+@import uk.gov.hmrc.http.HeaderCarrier
+@import uk.gov.hmrc.ptafrontend.viewmodels.attorneybanner.AttorneyBanner
+@import uk.gov.hmrc.ptafrontend.views.html._
+@import uk.gov.hmrc.sca.config.AppConfig
+@import uk.gov.hmrc.sca.models.BannerConfig
+@import uk.gov.hmrc.sca.views.html.components.FullWidthMainContent
+
+@this(
+        hmrcLayout: HmrcLayout,
+        appConfig: AppConfig,
+        govukBackLink: GovukBackLink,
+        hmrcHead: HmrcHead,
+        hmrcStandardHeader: HmrcStandardHeader,
+        hmrcStandardFooter: HmrcStandardFooter,
+        hmrcStandardPage: HmrcStandardPage,
+        hmrcTrackingConsentSnippet: HmrcTrackingConsentSnippet,
+        hmrcLanguageSelect: HmrcLanguageSelect,
+        hmrcTimeoutDialog: HmrcTimeoutDialog,
+        hmrcReportTechnicalIssueHelper: HmrcReportTechnicalIssueHelper,
+        hmrcScripts: HmrcScripts,
+        alphaBanner: StandardAlphaBanner,
+        betaBanner: StandardBetaBanner,
+        hmrcLanguageSelectHelper: HmrcLanguageSelectHelper,
+        hmrcTimeoutDialogHelper: HmrcTimeoutDialogHelper,
+        hmrcAccountMenu: HmrcAccountMenu,
+        ptaScripts: PtaScripts,
+        ptaHead: PtaHead,
+        govukPhaseBanner: GovukPhaseBanner,
+        ptaAttorneyBanner: PtaAttorneyBanner,
+        twoThirdsOneThirdMainContent: TwoThirdsOneThirdMainContent,
+        fullWidthMainContent: FullWidthMainContent,
+        twoThirdsMainContent: TwoThirdsMainContent
+)
+
+@(menu: HtmlFormat.Appendable,
+        serviceNameKey: Option[String],
```

Comment:
Do we want to make our parameter lists more closely match HmrcStandardLayout if we're just "wrapping" it?